subsystems/mvdm/ntvdm/bios/bios32/vbe.c

```diff
@@ -646,7 +646,7 @@ VOID WINAPI VbeService(LPWORD Stack)
 
                 default:
                 {
-                    DPRINT("VESA INT 0x10, AL = 0x05, Unknown subfunction: %02X", getBH());
+                    DPRINT("VESA INT 0x10, AL = 0x05, Unknown subfunction: 0x%02X\n", getBH());
```

Comment:
Personally I do like your commits adding the missing \n.
One of my rules for backporting though, is to avoid increasing the binary size at all costs.
Therefore the following suggestion(s).
```suggestion
                    DPRINT("VESA INT 0x10, AL = 0x05, Unknown subfunc: 0x%02X\n", getBH());
```

Reply:
I had some ideas for string changes myself, but I did minimal changes on purpose.
Unless an author/maintainer agrees with you, I'll stick to that.

Reply:
1. Having a debug log where the _**actual authors**_ of the code written (aka. Aleksander and me) can understand the debug output, without obscure abbreviations, is preferable.
2. This DPRINT was not compiled since it's DPRINT (not DPRINT1) and NDEBUG is defined.

@SergeGautherie Just keep your original modification (0x and \n)

Reply:
"obscure abbreviations"
That argumentation is absurd. Not shortening them will make me not porting the changes back, which is even worse.
There is nothing obscure with "max" and "func".

Reply:
These are uncompiled DPRINTs so why are you complaining? The binary size won't change.

Reply:
They can be improved. That is the point. And some of the prints we do discuss here are DPRINT**1**

Reply:
(A couple notes of mine, not to be discussed here.)

I have no idea what rules are for backporting.

> One of my rules for backporting though, is to avoid increasing the binary size at all costs.

Such a rule looks somewhat odd to me: any fix that adds code (not text) could be blocked too...

> Not shortening them will make me not porting the changes back, which is even worse.

I would assume you can tweak such text as you want on released branches.